Sources/ApolloCodegenLib/Frontend/GraphQLValue.swift

```diff
@@ -42,7 +42,22 @@ extension GraphQLValue: JavaScriptValueDecodable {
       }
       self = .list(.fromJSValue(value, bridge: bridge))
     case "ObjectValue":
-      self = .object(.fromJSValue(jsValue["value"], bridge: bridge))
+      let value = jsValue["value"]
+
+      /// The JS frontend does not do value conversions of the default values for input objects,
+      /// because no other compliation is needed, these are passed through as is from `graphql-js`.
```

Comment:
```suggestion
      /// because no other compilation is needed, these are passed through as is from `graphql-js`.
```
Assuming this should say compilation?